wallet/main.cpp

```diff
@@ -528,11 +542,8 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CTransaction& tx, bool* pfMissingInput
 
         // Check against previous transactions
         // This is done last to help prevent CPU exhaustion denial-of-service attacks.
-        if (!tx.ConnectInputs(*txdb, mapInputs, mapUnused, CDiskTxPos(1, 1),
-                              boost::atomic_load(&pindexBest), false, false)) {
-            return error("AcceptToMemoryPool : ConnectInputs failed %s",
-                         hash.ToString().substr(0, 10).c_str());
-        }
+        TRYV(tx.ConnectInputs(mapInputs, mapUnused, CDiskTxPos(1, 1), boost::atomic_load(&pindexBest),
```

Comment:
Will a `AcceptToMemoryPool : ConnectInputs failed` or similar error still be thrown here?

Reply:
Since we're using TRYV/TRY, the error will be taken from inside that function and passed through the return value of this function.